### ISSUE TITLE
Re-train the last layers of a pre-trained model

### ISSUE THREAD

**Issue by Romainpaulus** [BVLC/caffe#637]
I have followed [this tutorial](http://caffe.berkeleyvision.org/feature_extraction.html) to extract features at the fc7 layer of the pre-trained alexNet, and I want to re-use these features to train the last fully-connected layers of alexNext. I have tried to follow [this tutorial](http://caffe.berkeleyvision.org/imagenet_training.html) and used the newly extracted features instead of the train and val leveldb, removed all the layers up to fc7, changed the type of fc7 to DATA with the correct source, and the network is running but not training anything (loss function down to 0 after the first iteration). I suspect that the extracted features do not include labels (I have included them in the first tutorial, they are not all equal to 0), I guess that's why training is not working.

Is this the correct way to re-train the last layers of a pre-trained network? If yes, how can I include labels to the extracted features? If not, what is the correct way to do this?
